src/Symfony/Bridge/Twig/Extension/FormExtension.php

```diff
@@ -23,7 +23,10 @@
  */
 class FormExtension extends \Twig_Extension implements \Twig_Extension_InitRuntimeInterface
 {
-    private $renderer;
+    /**
+     * Make this property private in 4.0.
+     */
+    public $renderer;
```

Comment:
I'd indeed throw an exception. And yeah, `__isset` and `__unset` would have to be implemented, so… not sure it's worth the hassle.